logstash-core-event/lib/logstash/event.rb

```diff
@@ -98,7 +98,13 @@ def clone
       copy[k] = begin v.clone rescue v end
```

Comment:
We should really use `LogStash::Util.deep_clone(@data)` because otherwise we get `TypeError: can't clone Fixnum` type errors.

This applies to `@metadata` too.